h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -109,6 +110,11 @@ public AlgorithmValuesProvider() {
     @API(help = "Seed for random numbers; passed through to the metalearner algorithm. Defaults to -1 (time-based random number)", gridable = true)
     public long seed;
 
+    @API(help = "Size of training subsample used to compute training metrics. Use -1 to use the whole training frame.",
+            level = API.Level.secondary,
+            direction = API.Direction.INOUT)
+    public long training_scoring_subsample_size;
+
```

Comment:
wondering if it's worth exposing this to the API, really.
If user is not happy with the sample metric, then he can just run the following afterwards:
```
se.predict(train)  # to get metric on the entire frame
se.predict(train.split_frame([0.5])[0])  # to get metric on half of it
```
to me, not a useful addition to the API. Even the complicated name suggests "don't use me".
@erin, any opinion?

Reply:
@sebhrusen I am ok either way, though (unlike AutoML), I don't worry about having too many parameters with Stacked Ensemble.  I wonder if we should tweak the default sample size (to something lower than 10,000?), or start with 10k and see if it's scalable.

Reply:
10k looks reasonable, the goal with such a fix is to prevent waste of computation time for something not really useful like `training_metrics`.